e2e/jasmine/BUILD.bazel

```diff
@@ -1,15 +1,29 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "copy_to_bin")
 load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
 
 jasmine_node_test(
     name = "test",
     srcs = ["test.spec.js"],
 )
 
+copy_to_bin(
+    name = "jasmine_shared_env_test.spec",
+    srcs = ["jasmine_shared_env_test.spec.js"],
+)
+
+copy_to_bin(
+    name = "jasmine_shared_env_bootstrap",
+    srcs = ["jasmine_shared_env_bootstrap.js"],
+)
+
 jasmine_node_test(
     name = "shared_env_test",
-    srcs = ["jasmine_shared_env_test.spec.js"],
-    bootstrap = ["e2e_jasmine/jasmine_shared_env_bootstrap.js"],
-    data = ["jasmine_shared_env_bootstrap.js"],
+    srcs = ["jasmine_shared_env_test.spec"],
+    data = ["jasmine_shared_env_bootstrap"],
+    templated_args = [
+        "--nobazel_patch_module_resolver",
```

Comment:
was this needed along with the --require option?

Reply:
backed this out